SmartDeviceLink/SDLStreamingVideoLifecycleManager.h

```diff
@@ -138,7 +139,16 @@ NS_ASSUME_NONNULL_BEGIN
  @param configuration The configuration of this streaming media session
  @return A new streaming manager
  */
-- (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connectionManager configuration:(SDLStreamingMediaConfiguration *)configuration NS_DESIGNATED_INITIALIZER;
+- (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connectionManager configuration:(SDLStreamingMediaConfiguration *)configuration __deprecated_msg("Use initWithConnectionManager:streamingMediaConfiguration:lifecycleConfiguration: instead");
```

Comment:
This file / class is private, so it's not necessary to deprecate a method here. You can just update this method and remove the old one.

Reply:
Fixed